### PR TITLE
sign in, log out, token refresh

### PR DESCRIPTION
**Pull request by nadyrbek97**
https://trello.com/c/PU0MuE2Z/9-sing-in-page

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
     "formik": "^2.2.9",
+    "history": "^5.3.0",
```
Comment (seregaskirev) [inline frontend/package.json:24]: let's use history from react-router

https://reactrouter.com/docs/en/v6/hooks/use-navigate

File: frontend/src/store/models/user.ts [hunk 2]
```diff
@@ -42,5 +51,61 @@ export const user = createModel<RootModel>()({
...
+    async signOut() {
+      const accessToken = getAccessToken();
+      await http.post(
+        SIGN_OUT_URL,
+        {},
+        {
+          headers: {
+            Authorization: `Bearer ${accessToken}`,
+          },
+        }
```
Comment (seregaskirev) [inline frontend/src/store/models/user.ts]: I think we should write wrapper for request to write it only one time

**Review by seregaskirev — CHANGES_REQUESTED** [review]

File: frontend/src/store/models/user.ts (hunk 2 above)
Comment (nadyrbek97) [reply frontend/src/store/models/user.ts]: yeah, you are right

File: frontend/package.json (hunk 1 above)
Comment (nadyrbek97) [reply frontend/package.json:24]: yeah, I checked that and noted that we could not use it outside component (inside axios interceptor), I will try find another solution than :)

**Review by seregaskirev — APPROVED** [review]
👍

**Pull request merged**